.gitignore

```diff
@@ -28,4 +28,6 @@ yarn.lock
 ._.DS_Store
 */.DS_Store
 **/.DS_Store
-**/._.DS_Store
\ No newline at end of file
+**/._.DS_Store
+package.json
+package-lock.json
```

Comment:
um these 2 shouldn't be in gitignore!
they're supposed to be shared

Reply:
Fixed in commit b6c83a4